app/routes/photoTagRoutes.js

```diff
@@ -1,11 +1,12 @@
 import express from "express";
+import PhotoTagController from "../controllers/photoTagController.js";
 
 const photoTag = express.Router();
 
-photoTag.route("/create"); //create a phototag
+photoTag.route("/create").post(PhotoTagController.create); //create a phototag
 photoTag.route("/tag-name/:tagName"); //get a tag by its name
 photoTag.route("/filter"); //get photo tags by colors, creating users
 photoTag.route("/update/:tagName"); //update a photo tag
-photoTag.route("/delete/:tagName"); //delete a photo tag
+photoTag.route("/delete/:tagName").delete(PhotoTagController.delete); //delete a photo tag
```

Comment:
The ticket mentions this, but only Admins/Photographers should be allowed to delete (and create) photo tags.  Use your discretion when deciding who should be authorized, and work it into your tests